docs/developers/library_examples.md

```diff
@@ -0,0 +1,614 @@
+# Library Examples
+
+## C
+
+Coming Soon ...
+
+## C++
+
+### client_connection
+
+Available Since: v4.2.9
```

Comment:
Any advantage to linking to Doxygen pages from here? Is that even possible?

Reply:
Sounds like a good idea to me.

Reply:
yes, you can link to it, same as in the sidebar...    if you need something more targeted, then I'm not sure how stable the generated URLs are over time/versions.

Reply:
I'll add a link to the doxygen documentation at the top of the page. Like @trel said, it may be troublesome to manage links if they change between versions. We can also revisit this later.

Reply:
True. I've updated the text to link against things that are stable.